docs/migration-guides/1.10-1.11.md

```diff
@@ -0,0 +1,54 @@
+# Migrate from v1.10 to v1.11
+
+## Knative Eventing Mesh 
+
+To use new [Knative Eventing Mesh](https://kyma-project.io/docs/master/components/knative-eventing-mesh/#overview-overview), you must upgrade Kyma from version 1.10 to version 1.11. To do so, follow the [Kyma upgrade guide](https://kyma-project.io/docs/#installation-upgrade-kyma). 
+
+**CAUTION**: Events sent during the migration can be lost.
+    
+### Verification
+
+1. List all the applications:
+
+    ```bash
+     kubectl get applications 
+    ```  
+2. Check if each Application has a corresponding HTTP Source Adapter. 
+
+    ```bash
+    kubectl -n kyma-integration get pods 
+    ```
+3. Check if the Channel status is `READY`
+
+4. Check if Kyma Subscriptions have been converted to Knative Triggers:
+
+    ```bash
+    kubectl -n test get triggers 
+    ``` 
+    **NOTE**: If any of the conditions above was not fulfilled, recreate the Application using the Kyma Console. If the           problem still exists, create a support ticket.
+
+5. If you added a service using an EventsServiceClass, check if the service instance is present in your Namespace:
+    
+    ```bash
+    kubectl -n test get serviceinstances.servicecatalog.k8s.io 
+    ```
+  
+6. Check if a Broker exisits in your Namespace:
+    
+    ```bash
+       kubectl -n test get brokers
+    ```
+    
+7. Check if a Knative Subscription exists in the `kyma-integration` Namespace, linking the Knative Channel to the Knative        Broker: 
+
+    ```bash
+    kubectl -n kyma-integration get subscriptions.messaging.knative.dev 
+    ```
+8. Check if a subscriber URI in the Knative Subscription points to the Broker:
+
+    ```bash
+    kubectl -n kyma-integration get subscriptions.messaging.knative.dev brokersub-skqcq -o jsonpath='{ .spec.subscriber.uri }' 
```

Comment:
```suggestion
    kubectl -n kyma-integration get subscriptions.messaging.knative.dev -lapplication-name -o jsonpath='{ .spec.subscriber.uri }' 
```

Reply:
`-lapplication-name` is not valid.
the labels provided are:
```
    "labels": {
      "application-name": "test",
      "broker-namespace": "test"
    },
```
Where the applcation name actually is application name is actually `newapp` not `test`!
Any suggestions?

Reply:
```kubectl -n kyma-integration get subscriptions.messaging.knative.dev -o jsonpath='{ .items[*].spec.subscriber.uri }' -lapplication-name```